apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -369,31 +366,9 @@ public void trimLoadedFiles() {
             });
   }
 
-  /**
-   * Tests that synthetic data can be loaded.
-   *
-   * <p>TODO: Why is this disabled?
-   */
-  @Disabled
-  @Test
-  public void buildSyntheticLoadedFiles() {
-    PipelineTestUtils.get()
-        .doTestWithDb(
-            (dataSource, entityManager) -> {
-              loadSample(Arrays.asList(StaticRifResourceGroup.SYNTHETIC_DATA.getResources()));
-              // Verify that a loaded files exsits
-              final List<LoadedFile> loadedFiles =
-                  PipelineTestUtils.get().findLoadedFiles(entityManager);
-              assertTrue(loadedFiles.size() > 0, "Expected to have at least one file");
-              final LoadedFile file = loadedFiles.get(0);
-              final List<LoadedBatch> batches = loadBatches(entityManager, file.getLoadedFileId());
-              assertTrue(batches.size() > 0);
-            });
-  }
-
   /** Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_U} data. */
-  @Test
   @Disabled
```

Comment:
Lets make sure this gets annotated with a description; we shouldnt have any disabled left without an explanation.